### DISCUSSION TITLE
Getting a 400 when trying to transcribe a pre-recorded wav audio file

### DISCUSSION THREAD

**Discussion by alikaragoz** [deepgram/community#205] [General help] [answered]
### Which Deepgram product are you using?

Deepgram API

### Details

I'm trying to transcribe a pre-recorded audio and I'm getting a `400`.
On the documentation page `https://developers.deepgram.com/reference/pre-recorded` I'm just uploading a `.wav` file and trying to perform a `Curl` request via the console on the right.

I also tried with one of your sample `.wav` file and the result is the same.
I get pretty much the same result when trying from my Swift code.

Can you provide any guidance on what I might be doing wrong?

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

https://api.deepgram.com/v1/listen

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

a01f1229-7fc5-463a-b690-09b5fc33427b

### If possible, please attach your code or paste it into the text box.

```
curl --request POST \
     --url 'https://api.deepgram.com/v1/listen' \
     --header 'Authorization: Token MY_API_KEY' \
     --header 'accept: application/json' \
     --header 'content-type: application/json' \
     --data 'data:audio/wav;name=samples_jfk.wav;base64,NOT_PASTED_BINARY_STRING'
```

**Comment by nikolawhallon** [comment, marked as answer]
Can you try:

```
curl -X POST \
     "https://api.deepgram.com/v1/listen" \
     -H "Authorization: Token MY_API_KEY" \
     --data-binary @samples_jfk.wav
```

What you have, sending JSON, is the right method when asking Deepgram to grab a file from a URL to transcribe, e.g.:
```
curl \
  -X POST \
  "https://api.deepgram.com/v1/listen" \
  -H "Authorization: Token MY_API_KEY" \
  -H "content-type: application/json" \
  -d '{"url":"https://static.deepgram.com/examples/Bueller-Life-moves-pretty-fast.wav"}'
```

but for directly uploading the audio in the request body, it can/should be sent as binary and not JSON.

**Reply by alikaragoz** [reply]
Hey @nikolawhallon 👋 
I appreciate your help, removing the `content-type` did the trick.
The documentation needs to be updated IMO, it's a bit misleading.

**Reply by nikolawhallon** [reply]
I like the idea of having both JSON body and binary body examples somehow side-by-side in the docs myself, we'll figure something out! Thanks for catching this.

**Comment by lukeocodes** [comment]
Hey @alikaragoz thanks for your help. I agree that we need to improve the API reference on this. We also have a [tips and tricks guide](https://developers.deepgram.com/docs/transcribing-pre-recorded-audio#transcribe-a-local-file) which appears to have the correct format.
